src/components/moonlightsettingsview/resolution/resolutionsection.tsx

```diff
@@ -19,6 +19,18 @@ export const ResolutionSection: FC<Props> = ({ hostSettings }) => {
   return (
     <DialogControlsSection>
       <DialogControlsSectionHeader>Resolution</DialogControlsSectionHeader>
+      <Field
```

Comment:
After the review, I think that video codec does not fit in the `HostResolution` struct since it's not the default value for `class Dimension(TypedDict)`. I think it should go to `class HostSettings(TypedDict)` until there is a better place for it.

This field then should also go to the general section for now.